integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/util/TestDatabaseName.java

```diff
@@ -0,0 +1,36 @@
+package com.datastrato.gravitino.integration.test.util;
+
+/**
+ * An enum representing the different test database names used for testing purposes in the Gravitino
+ * project.
+ *
+ * <p>This enum provides a set of predefined database names that can be used in test cases to
+ * specify the target MySQL database for testing various functionalities and components of
+ * Gravitino.
+ *
+ * <p>The available test database names are:
+ *
+ * <ul>
+ *   <li>{@link #MYSQL_JdbcBackend}: Represents the MySQL database used for testing the JDBC backend
+ *       of Gravitino.
+ *   <li>{@link #MYSQL_MysqlAbstractIT}: Represents the MySQL database used for testing the
+ *       MysqlAbstractIT and its subclasses.
+ *   <li>{@link #MYSQL_AuditCatalogMysqlIT}: Represents the MySQL database used for testing the
+ *       AuditCatalogMysqlIT.
+ *   <li>{@link #MYSQL_CatalogMysqlIT}: Represents the MySQL database used for testing the catalog
+ *       integration with MySQL.
+ * </ul>
+ */
+public enum TestDatabaseName {
+  /** Represents the MySQL database used for JDBC backend of Gravitino. */
+  MYSQL_JdbcBackend,
```

Comment:
It's advised to use upper case value as the name of the enum name.

Reply:
like `MYSQL_JDBCBACKEND` ?

Reply:
Yeah.

Reply:
I think `MYSQL_JDBC_BACKEND` is more human readable.

Reply:
It's OK also.

Reply:
So it will become:

```
public enum TestDatabaseName {
  MYSQL_JDBC_BACKEND,
  MYSQL_MYSQL_ABSTRACT_IT,
  MYSQL_AUDIT_CATALOG_MYSQL_IT,
  MYSQL_CATALOG_MYSQL_IT
}
```